libs/payments/paypal/src/lib/paypal.manager.spec.ts

```diff
@@ -33,4 +42,51 @@ describe('paypalManager', () => {
   it('instantiates class (TODO: remove me)', () => {
     expect(paypalManager).toBeTruthy();
   });
+
+  describe('getCheckoutToken', () => {
+    const token = faker.string.uuid();
+
+    const successfulSetExpressCheckoutResponse =
+      NVPSetExpressCheckoutResponseFactory({
+        TOKEN: token,
+      });
+
+    const validOptions = { currencyCode: faker.finance.currencyCode() };
+
+    it('returns token', async () => {
+      paypalClient.setExpressCheckout = jest
+        .fn()
+        .mockResolvedValueOnce(successfulSetExpressCheckoutResponse);
+
+      const result = await paypalManager.getCheckoutToken(validOptions);
+      expect(result).toEqual(successfulSetExpressCheckoutResponse.TOKEN);
+    });
```

Comment:
[non-blocking] This can probably be merged with the `calls setExpressCheckout with passed options` test below to shorten things up a bit